src/app/global.js

```diff
@@ -51,5 +51,7 @@ var
     // Torrent engines
     peerflix = require('peerflix'),
 
+    WebTorrent = require('webtorrent');
+
```

Comment:
it looks like you could just import webtorrent in streamer.js doesn't it, @vankasteelj is there anything against just doing that ?